src/components/Videos.vue

```diff
@@ -123,6 +125,23 @@ export default {
 			// reset and show poster after play
 			this.$el.autoplay = false
 			this.$el.load()
+		},
+
+		saveVolume() {
+			localStorage.videoVolume = this.$el.volume
```

Comment:
localStorage is not available on all browsers! :)
We should have a check and a fallback if possible

Reply:
Thanks for the comment!

Yes, a check here should be necessary.